src/Game.h

```diff
@@ -99,6 +99,17 @@ class Game {
 	void SetTimeAccel(TimeAccel t);
 	void RequestTimeAccel(TimeAccel t, bool force = false);
 
+    /**
+     * Requests an increase in time acceleration
+     * @param force if set to false the system can reject the request under certain conditions
+     */
+    void RequestTimeAccelInc(bool force = false);
```

Comment:
We have Doxygen support (see COMPILING.txt) in pioneer.

As for comment style, I'll have to ping @johnbartholomew and @fluffyfreak as I'm not sure what the official style is.